lib/collection/src/shards/transfer/mod.rs

```diff
@@ -107,7 +108,9 @@ impl ShardTransferKey {
 }
 
 /// Methods for transferring a shard from one node to another.
-#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, Default, Serialize, Deserialize, JsonSchema)]
+#[derive(
+    Debug, Clone, Copy, Hash, PartialEq, Eq, Default, Serialize, Deserialize, JsonSchema, Anonymize,
+)]
```

Comment:
We skip anonymization for this type so we can remove the derive.

```suggestion
#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, Default, Serialize, Deserialize, JsonSchema)]
```